test/e2e/network/service.go

```diff
@@ -3350,7 +3350,7 @@ var _ = common.SIGDescribe("Services", func() {
 				Labels: testSvcLabels,
 			},
 			Spec: v1.ServiceSpec{
-				Type: "ClusterIP",
+				Type: "LoadBalancer",
```

Comment:
There is a trick @pacoxu , set the `spec.service.Spec.LoadBalancerClass` to something as `fakeClass` so the controllers on the cloud-provider skip it

Reply:
ooh, good idea